node-bootstrapper/app.go

```diff
@@ -100,6 +104,60 @@ func (a *App) provisionStart(ctx context.Context, cse utils.SensitiveString) err
 	return err
 }
 
+// usage example:
+// node-bootstrapper monitor
+func (a *App) Monitor(ctx context.Context) error {
+	timeoutCtx, cancel := context.WithTimeout(ctx, 15*time.Minute)
```

Comment:
probably fine for now, but we'll most likely need to make the timeout configurable

RP currently imposes a VM extension provisioning timeout which is dictated by a toggle

Reply:
removed timeout since RP already has timeout configured on cse and provisioning scripts have timeout baked in as well